connectors/camel-activemq-kafka-connector/pom.xml

```diff
@@ -118,6 +118,18 @@
         <enabled>true</enabled>
       </releases>
     </repository>
+    <!-- camel-ipfs and camel-weka -->
```

Comment:
I this repo not needed anymore with camel 3.2.0 or it wasn't ever needed?

Reply:
This repo seems to be needed from 3.2.0 ahead, because of weka and camel-ipfs